### ISSUE TITLE
Drag&Drop paste issue

### ISSUE THREAD

**Issue by bogusweb** [ngneat/input-mask#69]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior
 If u try select a value in other window or something then try to drag and drop this value in input with mask, this value is not pasted, but without inputMask library the value is pasted.

Angular v13 with @ngneat/input-mask@5.2.0 - https://stackblitz.com/edit/angular-ivy-vpnlko
Angular v11 with @ngneat/input-mask@4.1.1 - https://stackblitz.com/edit/angular-ivy-aq5gkw

## Expected behavior
Drag&Drop a value in input with mask should update formControl

## Minimal reproduction of the problem with instructions
![drag-drop-problem](https://user-images.githubusercontent.com/5169399/165396641-017ae566-3fbb-4dca-8992-f892deeb1d99.gif)

## Environment

<pre><code>
Angular version: X.Y.Z

Browser:
- [ ] Chrome (desktop) version XX
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
For Tooling issues:
- Node version: XX  
- Platform:  

Others:

</code></pre>

**Comment by shhdharmen** [comment]
How are you able to drag drop the text? Seems like you're using some extension.

**Comment by bogusweb** [comment]
nope, its normal behavior in windows, u just select text and u can drag&drop a selected text :D

**Comment by shhdharmen** [comment]
I don't have windows. Would you like to debug and provide fix?

**Comment by shhdharmen** [comment]
@bogusweb can you check if it's an issue with native library?

**Comment by bogusweb** [comment]
tested on site https://robinherbots.github.io/Inputmask/

![inputmask-drag-drop](https://user-images.githubusercontent.com/5169399/197419456-0be65b1f-1519-4a1e-a3d4-36e7a50c7a10.gif)
